examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -830,6 +830,14 @@
     "The behavior and performance of different optimizers depend strongly on the problem instance and, in particular, the problem size.  As we have seen above, analytical gradient estimation, such as employed in GD and QNG quickly become impractical when the system size gets larger, due to the increasing number tasks and shots that are required to calculate the gradient. Stochastic optimizers, such as QN-SPSA are designed to overcome this challenge. Let's take a look how QN-SPSA performs as we go to larger system sizes. In the following cells, we simulate a larger QAOA graph with 29 nodes and 58 edges. We use PennyLane's GPU simulator `lightning.gpu` for this workload, as it provides up to 10X speedup for intermediate scale problems (problems with 20-29 qubits). Before you run the cell, note that QN-SPSA takes 3.5 min per step, and about 3.5 hours overall to finish the algorithm."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "96364c00",
+   "metadata": {},
+   "source": [
+    "**Note:** The following cell may be unable to complete with the default resource limits. You may contact [AWS Support](https://support.console.aws.amazon.com/support/home#/) to increase the limits on your account."
```

Comment:
Should we be more specific the URL here? 

https://support.console.aws.amazon.com/support/home#/case/create?issueType=service-limit-increase